queries.sql

```diff
@@ -0,0 +1,24 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE
+      date_of_birth >= '2016-01-01'
+  and date_of_birth <= '2019-12-31';
+
+SELECT name FROM animals WHERE
+        neutered
+    and escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE
+       name = 'Agumon'
+    or name = 'Pikachu';
+
+SELECT name, escape_attempts FROM animals WHERE
+    weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered;
```

Comment:
Please select all animals where `neutered = 'true'` so the condition should be true for all animals where `neutered` has a boolean value `true`.

Reply:
the result of `neutered = 'true'` is the same as `neutered`